python_modules/libraries/dagstermill/dagstermill/asset_factory.py

```diff
@@ -95,8 +97,8 @@ def define_dagstermill_asset(
             contains letters, numbers, and _) and may not contain python reserved keywords.
         ins (Optional[Mapping[str, AssetIn]]): A dictionary that maps input names to information
             about the input.
-        non_argument_deps (Optional[Union[Set[AssetKey], Set[str]]]): Set of asset keys that are
-            upstream dependencies, but do not pass an input to the asset.
+        deps (Optional[Sequence[Union[AssetsDefinition, SourceAsset, AssetKey, str]]]): The assets
+            that are upstream dependencies, but do not pass an input value to the asset.
```

Comment:
> pass an input value to the asset

"an input value to the assets compute function" or something? "that are used in the notebook" ?  I think its worth pointing at something more specific than "the asset"